test/e2e/containers_conf_test.go

```diff
@@ -81,11 +83,17 @@ var _ = Describe("Podman run", func() {
 
 	It("podman Capabilities in containers.conf", func() {
 		os.Setenv("CONTAINERS_CONF", "config/containers.conf")
+		if IsRemote() {
+			podmanTest.RestartRemoteService()
+		}
```

Comment:
This is probably just ginkgo-ignorance on my part, but: given that `BeforeEach()` already does this (Setenv,Restart), why is it necessary in this case, and lines 104-107, 118-121, and 132-135? (93-97 is correct because it changes the envariable).

Reply:
It most likely is not.

Reply:
Removed